pdns/syncres.hh

```diff
@@ -120,10 +120,23 @@ public:
     return (unsigned int)d_cont.size();
   }
 
+  typedef std::vector<boost::tuple<ComboAddress,DNSName,uint16_t> > throttles_t;
+  throttles_t getThrottleTuples() {
```

Comment:
I'm not sure we need to do a full copy there, we could probably get away with returning a const reference to the actual map, and it would give us the content of the map in addition to the keys.